src/frontend/src/lib/components/core/LoadersGuard.svelte

```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+	import type { ComponentType } from 'svelte';
+	import Loaders from '$lib/components/core/Loaders.svelte';
+	import NoLoaders from '$lib/components/core/NoLoaders.svelte';
+	import { authSignedIn } from '$lib/derived/auth.derived';
+
+	let cmpLoaders: ComponentType;
+	$: cmpLoaders = $authSignedIn ? Loaders : NoLoaders;
+</script>
+
+<svelte:component this={cmpLoaders}>
```

Comment:
out of curiosity: does it work to do directly `this={$authSignedIn ? Loaders : NoLoaders}` ?

Reply:
It seems so yes. I did not know, I had to had it a try.